src/modules/crud/crud-saga.js

```diff
@@ -0,0 +1,29 @@
+import { addUser, fetchUsers } from "modules/users/user-effects";
+import { user, users } from "modules/entities/entities-schema";
+import { USER, USER_LIST } from "modules/crud/crud-entities";
+
+export const mapEntityToSaveParams = (entity, isUpdate) => {
+  switch (entity) {
+    case USER:
+      return {
+        effect: addUser,
+        schema: user
+      };
+    default:
+      return {};
```

Comment:
This is actually not necessary but it's not mentioned in the docs so it's not your fault.